python/mxnet/contrib/quantization.py

```diff
@@ -408,12 +420,23 @@ def _load_params(params, logger=logging):
         raise ValueError('Unsupported params provided. Must be either a path to the param file or'
                          ' a pair of dictionaries representing arg_params and aux_params')
 
+def save_params(fname, arg_params, aux_params, logger=None):
```

Comment:
This is considered as a public API under the contrib.quantization namespace. What is the necessity of adding such an API?

Reply:
This is already inside imagenet_gen_qsym.py. Will remove it.